apps/climatemappedafrica/src/components/Hero/Hero.js

```diff
@@ -0,0 +1,186 @@
+import { RichTypography } from "@commons-ui/core";
+import { Box, Grid, useMediaQuery } from "@mui/material";
+import { useTheme } from "@mui/material/styles";
+import dynamic from "next/dynamic";
+import PropTypes from "prop-types";
+import React, { useState } from "react";
+
+import heroBg from "@/climatemappedafrica/assets/images/bg-map-white.jpg";
+import DropdownSearch from "@/climatemappedafrica/components/DropdownSearch";
+import Image from "@/climatemappedafrica/components/Image";
+import RichHeader from "@/climatemappedafrica/components/RichHeader";
+import Section from "@/climatemappedafrica/components/Section";
+
+const Map = dynamic(() => import("./Map"), { ssr: false });
+
+function Hero({
+  comment,
+  title,
+  subtitle,
+  searchLabel,
+  featuredLocations,
+  searchPlaceholder,
+  properties,
+  location: { center },
+  level,
+  ...props
+}) {
+  const theme = useTheme();
+  const { typography, breakpoints } = theme;
+  const isUpLg = useMediaQuery(theme.breakpoints.up("lg"));
```

Comment:
As we've already injected `theme` via `ThemeProvider`, we can just:
```suggestion
  const isUpLg = useMediaQuery((theme) => theme.breakpoints.up("lg"));
```